02_classify_mutations/nbconverted/plot_expression_gene_sets.py

```diff
@@ -172,8 +169,10 @@
                           SIG_ALPHA,
                           metric=plot_metric,
                           verbose=True,
-                          # TODO: fix overlap for merged cancer genes
-                          mark_overlap=(not merged_geneset))
+                          label_x_lower_bounds=[0.2, 0.2, 0.4],
```

Comment:
Since you're making figures for the paper, these numbers aren't likely to change at this point, but it's probably worth a note about how you chose the lower bounds for x and y, since they seem to be slightly different for expression, methylation, and plot_all_results.